src/main/java/com/example/usermanageapirest/application/controller/request/UserCreateRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.example.usermanageapirest.application.controller.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Data
+@NoArgsConstructor
+@Schema(name = "CreateUserRequest", required = true)
+public class UserCreateRequest {
+
+	@JsonProperty("firstname")
```

Comment:
la nomenclatura json sigue camelCase igual q java

```suggestion
	@JsonProperty("firstName")
```